frontend/src/components/InputDialogs/InputDialogs.js

```diff
@@ -180,88 +179,84 @@ const InputDialogs = () => {
 
   function handleReadIn (e) {
     const input = e.target.value.toString()
-    setRead(input[input.length - 1] ?? '')
+    const value = input[input.length - 1]
+    setRead(value)
   }
+
   function handleWriteIn (e) {
     const input = e.target.value.toString()
-    setWrite(input[input.length - 1] ?? '')
+    const value = input[input.length - 1]
+    setWrite(value)
```

Comment:
Not to sure how I triggered it but got a null access error because of this change. Think using the null coalescing is best idea unless there was a reason you changed it